src/main/resources/admin/tools/rewrite-manager/rewrite-manager.xml

```diff
@@ -4,5 +4,6 @@
   <description>Rewrite-configuration and management</description>
   <allow>
     <principal>role:system.admin</principal>
+    <principal>role:com.enonic.xp.rewrite.admin</principal>
```

Comment:
```suggestion
    <principal>role:com.enonic.app.rewrite.admin</principal>
```

We reserve `com.enonic.xp` for XP itself. Apps must not use this namespace. Could you please fix it to `com.enonic.app.rewrite.admin`

Notice `app` instead of `xp`

Reply:
Ah, I see. Thanks for the clarification! It should be fixed now.